.env.development

```diff
@@ -12,6 +12,6 @@ INFO_EMAIL=info@edx.org
 AUTHN_ALGOLIA_APP_ID=''
 AUTHN_ALGOLIA_SEARCH_API_KEY=''
 # ***** Links *****
-AUTHN_TOS_AND_HONOR_CODE_LINK='http://localhost:18000/honor'
-AUTHN_PRIVACY_POLICY_LINK='http://localhost:18000/privacy'
+TOS_AND_HONOR_CODE='http://localhost:18000/honor'
+PRIVACY_POLICY='http://localhost:18000/privacy'
```

Comment:
Why did we change this?

Reply:
I renamed this to make it consistent with the names of authn so that when we put these for prospectus, they become consistent with authn MFE as well.